src/app/static/src/js/utils/types.ts

```diff
@@ -3,6 +3,25 @@ export interface Parameter {
     value: string
 }
 
+export interface Instances {
+    database: string,
+    instance: string
+}
+
+export type Dict<V> =  {[index: number]: V}
+
```

Comment:
I can't see if these are being used anywhere. If not then let's remove to avoid confusion in the future.